tests/components/__snapshots__/context_menu.test.ts.snap

```diff
@@ -80,7 +80,7 @@ exports[`Standalone context menu tests Context Menu context menu simple renderin
         class="o-icon"
       >
         <polygon
-          fill="#000000"
+          fill="currentColor"
```

Comment:
I'm quite surprised to see this in a snapshot.
Shouldn't it be rendered with an actual color ?

Reply:
same